.github/workflows/ci-latest-release.yml

```diff
@@ -9,16 +9,38 @@ on:
       - "KubeArmor/**"
       - "protobuf/**"
       - ".github/workflows/ci-latest-release.yml"
+      - "pkg/**"
       - "!STABLE-RELEASE"
 
   create:
     branches:
       - "v*"
 
 jobs:
+  check:
+    name: Check what pkg were updated
+    if: github.repository == 'kubearmor/kubearmor'
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    outputs:
+      kubearmor: ${{ steps.filter.outputs.kubearmor}}
+      controller: ${{ steps.filter.outputs.controller }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dorny/paths-filter@v2
```

Comment:
Why are we needing this one? I think `working-directory` should suffice here. I am thinking it to be something like, if the conditions are satisfied to create a release then enter the `pkg/KubeArmorController` and then use `make docker-buildx`

Reply:
we are first checking if a pkg (kubearmor, controller) has been updated before making a release:
so with this workflow release will be like:
| pkg changed              | release         |
|--------------------------|-----------------|
| kubearmor                | kubearmor only  |
| controller               | controller only |
| kubearmor+ <br>  controller | release both    |